library/src/main/java/eu/kanade/tachiyomi/source/Source.kt

```diff
@@ -48,6 +48,22 @@ interface Source {
      */
     suspend fun getSearchFilters(): FilterList
 
+    /**
+     * Get a page with a list of manga.
+     *
+     * @since extensions-lib 1.6
+     * @param page the page number to retrieve.
+     */
+    suspend fun getPopularMangaList(page: Int): MangasPage = throw Exception("Stub!")
```

Comment:
Maybe?
```suggestion
    suspend fun getDefaultMangaList(page: Int): MangasPage = throw Exception("Stub!")
```

Reply:
how about `getMangaList` here and `getSearchMangaList` for the search one?

Reply:
Don't like that

Reply:
I do think `getMangaList` being the method for search isn't immediately intuitive.

Reply:
It would've made sense if we had listing instead of the popular/latest function